app/src/main/java/de/westnordost/streetcomplete/data/quest/VisibleQuestsSource.kt

```diff
@@ -72,6 +73,13 @@ class VisibleQuestsSource(
         }
     }
 
+    private val cache = SpatialCache(
+        16,
+        SPATIAL_CACHE_SIZE,
+        3000,
```

Comment:
Please add comments how you selected these values, for documentation purposes.

Reply:
done for all used values